apps/api/src/app/layouts/usecases/delete-layout/delete-layout.use-case.ts

```diff
@@ -34,14 +34,11 @@ export class DeleteLayoutUseCase {
       throw new ConflictException(`Layout with id ${command.layoutId} is being used so it can not be deleted`);
     }
 
-    await this.layoutRepository.deleteLayout(command.layoutId, layout._environmentId, layout._organizationId);
+    if (layout.isDefault) {
+      throw new ConflictException(`Layout is being used as your default layout, so it can not be deleted`);
+    }
 
-    const createLayoutChangeCommand = CreateLayoutChangeCommand.create({
-      environmentId: command.environmentId,
-      layoutId: command.layoutId,
-      organizationId: command.organizationId,
-      userId: command.userId,
-    });
```

Comment:
Did you delete this?

Reply:
Yes, it was not being used. You create this command inside the createChange function.

Reply:
Argh missed that. Good catch!